pkgs/by-name/dw/dwlb/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, writeText
+, pkg-config
+, wayland
+, wayland-protocols
+, pixman
+, fcft
+, conf ? null
+}:
+stdenv.mkDerivation {
+  pname = "dwlb";
+  version = "0-unstable-2024-05-17";
+
+  src = fetchFromGitHub {
+    owner = "kolunmi";
+    repo = "dwlb";
+    rev = "0daa1c1fdd82c4d790e477bf171e23ca2fdfa0cb";
+    hash = "sha256-Bu20IqRwBP1WRBgbcEQU4Q2BZ2FBnVaySOTsCn0iSSE=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ wayland wayland-protocols pixman fcft ];
+
+  outputs = [ "out" "man" ];
+
+  # Allow alternative config.def.h usage. Taken from dwl.nix.
```

Comment:
I have changed this style in new dwl:

https://github.com/NixOS/nixpkgs/pull/330473

instead of `conf`, use `configDefH`;  and use the  Boolean flag `withCustomConfigDefH` instead of testing for null.